src/state_manager/mod.rs

```diff
@@ -606,18 +626,21 @@ where
     ///
     /// For details, see the documentation for [`apply_block_messages`].
     ///
-    #[instrument(skip(self, tipset, callback))]
+    #[instrument(skip_all)]
     pub async fn compute_tipset_state<CB: 'static>(
         self: &Arc<Self>,
         tipset: Arc<Tipset>,
         callback: Option<CB>,
-    ) -> Result<CidPair, Error>
+        enable_tracing: bool,
+    ) -> Result<(CidPair, ComputeStateOutput), Error>
     where
         CB: FnMut(&Cid, &ChainMessage, &ApplyRet) -> Result<(), anyhow::Error> + Send,
     {
         let this = Arc::clone(self);
-        tokio::task::spawn_blocking(move || this.compute_tipset_state_blocking(tipset, callback))
-            .await?
+        tokio::task::spawn_blocking(move || {
```

Comment:
nit: why the extra curly braces? does not make it any more readable.

Reply:
this is rustfmt's choice because the line is longer